### ISSUE TITLE
Buildifier removes comments when merging else if into elif

### ISSUE THREAD

**Issue by aherrmann** [bazelbuild/buildtools#645]
Buildifier removes comments when merge nested `else: if:` into `elif:`. See the following diff:

``` diff
-        else:
-            # GHCi can load PIC static libraries
-            # (compiled with -fPIC and -fexternal-dynamic-refs).
-            # Prefer static libraries over dynamic libraries to reduce the risk
-            # of overflowing the MACH-O header size limit on MacOS.
-            if static_lib:
-                lib = static_lib
-            elif dynamic_lib:
-                lib = dynamic_lib
+        elif static_lib:
+            lib = static_lib
+        elif dynamic_lib:
+            lib = dynamic_lib
```

This occurs with `buildtools-0.22.0`.

**Comment by vladmos** [comment]
This has been fixed with #573, please try a newer version of buildtools.

**Comment by aherrmann** [comment]
@vladmos Thanks for the pointer. Indeed, I can confirm that this doesn't happen anymore with version 0.25.1
